packages/confirmation-modal/src/ConfirmationModal/ConfirmationModal.tsx

```diff
@@ -146,4 +159,8 @@ ConfirmationModal.propTypes = {
   variant: PropTypes.oneOf(Object.values(Variant)),
   requiredInputText: PropTypes.string,
   darkMode: PropTypes.bool,
+  // eslint-disable-next-line react/forbid-prop-types
+  confirmButtonProps: PropTypes.objectOf(PropTypes.any),
+  // eslint-disable-next-line react/forbid-prop-types
+  cancelButtonProps: PropTypes.objectOf(PropTypes.any),
```

Comment:
our usage of `PropTypes` feels pretty inconsistent that it makes me wonder if it's actually serving a purpose or if it makes sense to move towards typescript-only support. If we want to type these better, I think it makes sense to export the button proptypes from button similar to how we export the typescript `ButtonProps` interface. can we avoid adding the lint disable if it's just a warning to flag that it's something we should clean up/remove in the future?

Reply:
We can't do TS only support because not everyone using our packages use TS. However, i can remove the comments for now.